docs/src/basics/comments_and_logging.md

```diff
@@ -0,0 +1,37 @@
+# Comments and Logging
+
+## Comments
+
+Comments in Sway start with two slashes and continue until the end of the line. For comments that extend beyond a single line, you’ll need to include // on each line.
+
+```sway
+// hello world
+```
+
+```sway
+// let's make a couple of lines
+// commented.
+```
+
+You can also place comments at the ends of lines containing code.
+
+```sway
+fn main() {
+    let baz = 8; // Eight is a lucky number
+}
+```
+
+## Logging
+
+To log integers to stdout, you can use the `log_u64`, `log_u32`, `log_u16`, or `log_u8` functions from the `lib-std` library.
```

Comment:
Logging isn't like `console.log`. See https://blog.chain.link/events-and-logging-in-solidity/ for more info. This deserves its own page tbh, but can always be left to a subsequent PR.
```suggestion
To log integers, you can use the `log_u64`, `log_u32`, `log_u16`, or `log_u8` functions from the `lib-std` library.
```